flink/connectors/kafka.rst

```diff
@@ -0,0 +1,28 @@
+===============
+Kafka Connector
+===============
+
+Due to its nature, the distributed publish/subscribe Apache Kafka is one of the best matches to read data from and write data to for Flink.
+
+The `official documentation <https://ci.apache.org/projects/flink/flink-docs-release-1.0/apis/streaming/connectors/kafka.html>`_ contains in-depth details on the high level of interoperability between Kafka and Flink.
+
+To use the connector, you should first of all include the following dependency in your project's POM:
+
+.. literalinclude:: /rbp-docs-code/mvn/pom.xml
+    :caption: pom.xml
+    :language: xml
+    :lines: 124-128
+
+RBFDD includes a custom Flink version that allows the user to make the two systems interoperate even on *secure* clusters.
```

Comment:
I would introduce a _Security_ section here, to state better that connector can be used with and *without* Kerberos security.